ee/clickhouse/sql/session_recording_events.py

```diff
@@ -13,6 +13,7 @@
     team_id Int64,
     distinct_id VARCHAR,
     session_id VARCHAR,
+    window_id VARCHAR,
```

Comment:
Is it expected that old values will all be `''` after this?

Reply:
Sanity check - given this is not optional, please make sure that ingesting these still works if no window_ids are coming in.

Reply:
Yeah, from what I was reading, it sounds like using Nullable can hurt perf and just leaning on `''` is preferred (definitely open to other thoughts here).

For the `window_id` is null case, I verified that this is still working. Although I'm a bit confused about where `null` is being converted to `''`. Not sure if it's baked into clickhouse or kafka?

Reply:
It'd be on the CH side since it would look into the payload and not see this an just assume `''`